jetty-http2/http2-hpack/src/main/java/org/eclipse/jetty/http2/hpack/MetaDataBuilder.java

```diff
@@ -74,11 +74,13 @@ public void emit(HttpField field) throws HpackException.SessionException
     {
         HttpHeader header = field.getHeader();
         String name = field.getName();
+        if (name == null || name.length() == 0)
```

Comment:
What happens if name is all whitespace?

Reply:
I'm now checking for ALPHA, DIGIT, TCHAR and COLON